wagtailio/static/sass/components/_get-started.scss

```diff
@@ -0,0 +1,53 @@
+.get-started {
+    padding-bottom: 50px;
+
+    @include media-query(medium) {
+        padding-bottom: 100px;
+    }
+
+    @include media-query(large) {
+        padding-bottom: 200px;
+    }
+
+    &__heading {
+        grid-column: 2 / span 2;
+
+        @include media-query(medium) {
+            grid-column: 2 / span 3;
+            margin-bottom: 40px;
+        }
+
+        @include media-query(large) {
+            grid-column: 2 / span 5;
+            margin-bottom: 100px;
+        }
+    }
+
+    &__list {
```

Comment:
Note: I think you need to add a little padding (40px) here, there's a slight offset on the list when compared to the title's indent.
![image](https://user-images.githubusercontent.com/22497837/182809376-01077046-ac6e-4e9a-809e-a6845473dd3f.png)

Reply:
Good spot. Added this in and reduced the percentage padding so they don't get too squashed on a laptop sized screen (just before the medium brekapoint kicks in)